tests/checkpointing/test_legacy_checkpoints.py

```diff
@@ -24,9 +24,27 @@
 CHECKPOINT_EXTENSION = ".ckpt"
 
 
-# todo: add more legacy checkpoints :]
+# todo: add more legacy checkpoints :] for < v0.8
```

Comment:
Can you move this list to utilities and add a test to check current version to it. The test should fail if the current version hasn't been added.

Reply:
> Can you move this list to utilities

What do you mean, pls?

well, there is nothing such as the current version as current versions are always in development...